string vid
Status radar
Status lidar
Status camera
